x/observer/keeper/ballot.go

```diff
@@ -58,10 +73,41 @@ func (k Keeper) GetAllBallots(ctx sdk.Context) (voters []*types.Ballot) {
 
 // AddBallotToList adds a ballot to the list of ballots for a given height.
 func (k Keeper) AddBallotToList(ctx sdk.Context, ballot types.Ballot) {
-	list, found := k.GetBallotList(ctx, ballot.BallotCreationHeight)
+	list, found := k.GetBallotListForHeight(ctx, ballot.BallotCreationHeight)
 	if !found {
 		list = types.BallotListForHeight{Height: ballot.BallotCreationHeight, BallotsIndexList: []string{}}
 	}
 	list.BallotsIndexList = append(list.BallotsIndexList, ballot.BallotIdentifier)
 	k.SetBallotList(ctx, &list)
 }
+
+// ClearMaturedBallotsAndBallotList deletes all matured ballots and the list of ballots for a given height.
+// It also emits an event for each ballot deleted.
+func (k Keeper) ClearMaturedBallotsAndBallotList(ctx sdk.Context, maturityBlocksParam int64) {
```

Comment:
I think the naming is too verbose. Consider renaming this to `CleanupBallots(...)`

Reply:
Imo , CleanupBallots might be less verbose than necessary, I do not see any harm in having a very verbose function name though

Reply:
Yeah it's subjective. I prefer concise `DoStuff()` over `DoThisAndThenThis()`